OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -34,6 +38,590 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                                2 stubs between <locus target="#61"/> and <locus target="#62"/>
+                                            </item>
+                                            <item xml:id="q11" n="11">
+                                                <dim unit="leaf">4</dim>
+                                                <locus from="63r" to="67v"/>
+                                            </item>
+                                            <item xml:id="q12" n="12">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="68r" to="76v"/>
+                                            </item>
+                                            <item xml:id="q13" n="13">
+                                                <dim unit="leaf">8</dim>
+                                                <locus from="77r" to="84v"/>
+                                            </item>
+                                            <item xml:id="q14" n="14">
+                                                <dim unit="leaf">8</dim>
+                                                <locus from="85r" to="92v"/>
+                                                3, stub after 6
+                                                6, stub after 3
+                                            </item>
+                                            <item xml:id="q15" n="15">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="93r" to="101v"/>
+                                            </item>
+                                            <item xml:id="q16" n="16">
+                                                <dim unit="leaf">7</dim>
+                                                <locus from="102r" to="108v"/>
+                                            </item>
+                                            <item xml:id="q17" n="17">
+                                                <dim unit="leaf">6</dim>
+                                                <locus from="109r" to="114v"/>
+                                                stub between <locus target="#113"/> and <locus target="#114"/>
+                                            </item>
+                                            <item xml:id="q18" n="18">
+                                                <dim unit="leaf">9</dim>
+                                                <locus from="115r" to="123v"/>
+                                            </item>
+                                        </list>
+                                    </collation>
+                                    <condition key="deficient">
+                                        The manuscript is kept in a box, in an embroidered rectangular piece of textile, measuring 620 x 920 mm. <!-- how to encode this information? -->
```

Comment:
I think that the embroidered textile could be considered as an enclosure according to Ligatus definition https://www.ligatus.org.uk/lob/concept/3311, and added to binding description. However, this has never been encoded in BM. 
As regards the box, I think it is useful to mention it, in order to give information on the preservation state of the manuscript.